templates/bundles/NucleosProfileBundle/Registration/register_content.html.twig

```diff
@@ -10,10 +10,39 @@
       </a>
     </div>
     <hr>
+    {{ dump(form.vars.errors) }}
     {{ form_start(form, {'method': 'post', 'action': path('nucleos_profile_registration_register'), 'attr': {'class': 'nucleos_profile_registration_register'}}) }}
       {{ form_row(form.email) }}
       {{ form_row(form.username) }}
+      {% set break = false %}
+      {% for key, error in form.vars.errors if not break %}
+        {% if error.cause.propertyPath == 'username' %}
+        <div class="alert alert-danger">
+          <ul class="list-unstyled">
+            <li>
+              <span class="glyphicon glyphicon-exclamation-sign"></span>
+              {{ error.message }}
+            </li>
+          </ul>
+        </div>
+          {% set break = true %}
+        {% endif %}
+      {% endfor %}
```

Comment:
It doesn't, There are 2 problems
- Error bubbling cannot be activated as the form doesn't inherit, it is injected but also directly referenced from the vendor code. The configuration for validations seems to be not present and without inheriting, i don't know how can i configure error bubbling.
- Validation also is not set up in a way that we can validate the password's security and choose which validations are active for the email and username (we want to prevent the "already used" message from showing)